experimental/handoff-library/README.md

```diff
@@ -6,7 +6,7 @@ The library implements the additions to the Bot Framework SDK to support handoff
 
 Integrations with specific agent hubs are not part of the library. 
 
-The library is intto be merged in a future release of the Bot Framework SDK.
```

Comment:
There is definitely a spelling mistake here. However, the grammar seems odd here. Maybe it should say something like "The library will be merged in a future release " or "The library is going to be merged..."